iframe/style.css

```diff
@@ -327,9 +343,6 @@ body.stop h1 {
 
 body.slow h1,
 body.without h1 {
-  background: linear-gradient(#39ebfd, #6b74fb);
```

Comment:
Nice! Learned something new today. Thanks!

I went ahead and wrapped the related style rules in a `@supports` at-rule set.